src/routers/handlers/before_you_file_package_accounts/before.you.file.package.accounts.ts

```diff
@@ -12,8 +13,9 @@ export class BeforeYouFilePackageAccountsHandler extends GenericHandler {
         });
     }
 
-    execute (_req: Request, _res: Response): ViewModel<BaseViewData> {
-
+    execute (req: Request, _res: Response): ViewModel<BaseViewData> {
+        // Clear company number if has been set before, if login already.
+        deleteExtraDataCompanyNumber(req.session);
```

Comment:
Do we need this? Wouldn't this be covered in the deleting of the session on submit?